Cargo.toml

```diff
@@ -19,6 +19,7 @@ path = "src/upm/lib.rs"
 
 [features]
 # No features by default
+# default = [ "test_database" ]
```

Comment:
This change doesn't need to be here. `--features=test_database` can be specified on the `cargo build` and `cargo run` command lines when needed.

Reply:
Well, when testing I usually like the possibility of --test as an argument. It's an easy and available resource for manual testing. Of course in release it should not be present.

 I read somewhere it is possible to target debug or release only, something like `[profile.debug]`. How about using that?

P.S. Will check, should be able to find it.